assets/app/scripts/filters/resources.js

```diff
@@ -790,28 +790,26 @@ angular.module('openshiftConsole')
       return _.startCase(kind);
     };
   })
-  .filter('humanizeResourceType', function() {
+  .filter('humanizeQuotaResource', function() {
     return function(resourceType) {
       if (!resourceType) {
         return resourceType;
       }
 
       var nameFormatMap = {
-        'build': 'Build',
-        'buildconfig': 'Build Config',
-        'deployment': 'Deployment',
-        'deploymentconfig': 'Deployment Config',
-        'imagestream': 'Image Stream',
+        'cpu.limit': 'CPU (Limit)',
+        'cpu.request': 'CPU (Request)',
+        'memory.limit': 'Memory (Limit)',
+        'memory.request': 'Memory (Request)',
+        'openshift.io/imagesize': 'Image Size',
+        'openshift.io/imagestreamsize': 'Image Stream Size',
+        'openshift.io/projectimagessize': 'Project Image Size',
         'persistentvolumeclaims': 'Persistent Volume Claims',
-        'pod': 'Pod',
         'pods': 'Pods',
-        'project': 'Project',
-        'resourcequotas': 'Resource Quotas',
-        'replicationcontroller': 'Replication Controller',
         'replicationcontrollers': 'Replication Controllers',
+        'resourcequotas': 'Resource Quotas',
         'route': 'Route',
```

Comment:
@derekwaynecarr Can you quota routes? I didn't see it in the docs. I accidentally left this here.

If so, I assumes it should be `routes`, plural.